Homework_24/script.js

```diff
@@ -0,0 +1,72 @@
+
+let a = new Array(13, 22, 31, 4, 5, 16, 7, 8, 9, 10);
+
+
+function addition(a, b) {
+  if (typeof a === "number" || typeof b === "number") {
```

Comment:
why `||` ?
what if `a` will be number, but `b` will be `null`?

should be `&&`